alert-rules.md

```diff
@@ -596,11 +596,11 @@ This section gives the alert rules for the TiKV component.
 
 * Alert rule:
 
-    `sum(rate(tikv_thread_cpu_seconds_total{name="apply_worker"}[1m])) by (instance) > 1.8`
+    `max(rate(tikv_thread_cpu_seconds_total{name=~"apply_.*"}[1m])) by (instance) > 0.9`
 
 * Description:
 
-    The pressure on the apply Raft log thread is too high. It is often caused by a burst of writes.
+    The pressure on the apply Raft log thread is too high, and has approached or exceeded the limit of the `apply` thread. It is often caused by a burst of writes.
```

Comment:
```suggestion
    The apply Raft log thread suffers from high pressure and has approached or exceeded its limit. This is often caused by a burst of writes.
```
The subject of 'has approached or exceeded the limit of the `apply` thread' is 'The pressure', which is inappropriate.